Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM debian:buster
+FROM debian:bookworm-slim
 
-RUN apt-get update
-RUN apt-get install -y curl nano certbot
-RUN curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl"
-RUN chmod 755 kubectl && mv kubectl /usr/local/bin/
+RUN apt-get update && apt-get install -y curl vim nano certbot && \
```

Comment:
For a project like this, optimizing the Docker image like this isn't really necessary in my opinion.
It doesn't hurt anything either, so thank you anyways :-)